kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/dsl/MultiDeleteable.java

```diff
@@ -16,11 +16,47 @@
 
 package io.fabric8.kubernetes.client.dsl;
 
+import java.util.Arrays;
 import java.util.List;
 
-public interface MultiDeleteable<T> {
+public interface MultiDeleteable<T> extends ListDeletable<T> {
```

Comment:
```suggestion
public interface MultiDeletable<T> extends ListDeletable<T> {
```
I guess, without the `e` is the right version analogous to `ListDeletable`